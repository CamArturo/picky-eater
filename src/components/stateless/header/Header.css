header {
  height: 100px; }
  header img {
    width: 300px;
    height: 40px;
    margin: 30px 0 0 100px; }

/* Large Devices, Wide Screens */
/* Medium Devices, Desktops */
/* Small Devices, Tablets */
/* Extra Small Devices, Phones */
@media only screen and (max-width: 480px) {
  header {
    display: flex;
    align-items: center;
    justify-content: center; }
    header img {
      width: 300px;
      height: 40px;
      margin: 0;
      padding: 0 20px; } }
/* Custom, iPhone Retina */

/*# sourceMappingURL=Header.css.map */
