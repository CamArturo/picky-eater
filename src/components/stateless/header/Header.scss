header {
  height: 100px;
  display: flex;
  align-items: center;
  img {
    width: 300px;
    height: 40px;
    margin: 0 30px 0 100px;
  }
  .tag-line p {
    font-family: 'Mukta Vaani', sans-serif;
    font-size: 1.1rem;
    margin: 12px 0 0 0;
  }
}

/* Large Devices, Wide Screens */
@media only screen and (max-width: 1200px) {

}

/* Medium Devices, Desktops */
@media only screen and (max-width: 992px) {

}

/* Small Devices, Tablets */
@media only screen and (max-width: 768px) {
  header {
    padding-top: 10px;
    display: flex;
    align-items: center;
    justify-content: center;
    flex-wrap: wrap;
    text-align: center;
    .home {
      width: 100%;
    }
    .tag-line p {
      margin: 0;
    }
  }
}

/* Extra Small Devices, Phones */
@media only screen and (max-width: 480px) {
  header {
    padding-top: 10px;
    display: flex;
    align-items: center;
    justify-content: center;
    flex-wrap: wrap;
    text-align: center;
    img {
      width: 300px;
      height: 40px;
      margin: 0;
      padding: 0 20px;
    }
    .tag-line p {
      margin: 0;
    }
  }
}

/* Custom, iPhone Retina */
@media only screen and (max-width: 320px) {

}


