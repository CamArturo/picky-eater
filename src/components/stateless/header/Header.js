import React from 'react';
import logo from '../../../assets/picky-eater-logo.png';
import './Header.css';
import { NavLink } from 'react-router-dom';

const Header = () => {
  return (
    <header>
      <NavLink to="/" className="home"><img src={logo} className="App-logo" alt="Picky Eater Logo" /></NavLink>
    </header>
  );
};

export default Header;
