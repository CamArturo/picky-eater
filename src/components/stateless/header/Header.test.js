import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';


it('Should match the snapshot', () => {
  const appComponent = shallow(<Header />);

  expect(appComponent).toMatchSnapshot();
});
