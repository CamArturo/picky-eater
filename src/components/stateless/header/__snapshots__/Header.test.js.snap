// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="home"
          to="/"
>
          <img
                    alt="Picky Eater Logo"
                    className="App-logo"
                    src="picky-eater-logo.png"
          />
</NavLink>,
        <div
          className="tag-line"
>
          <p>
                    There is a right choice after all.
          </p>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <img
            alt="Picky Eater Logo"
            className="App-logo"
            src="picky-eater-logo.png"
/>,
          "className": "home",
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Picky Eater Logo",
            "className": "App-logo",
            "src": "picky-eater-logo.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            There is a right choice after all.
</p>,
          "className": "tag-line",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "There is a right choice after all.",
          },
          "ref": null,
          "rendered": "There is a right choice after all.",
          "type": "p",
        },
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="home"
            to="/"
>
            <img
                        alt="Picky Eater Logo"
                        className="App-logo"
                        src="picky-eater-logo.png"
            />
</NavLink>,
          <div
            className="tag-line"
>
            <p>
                        There is a right choice after all.
            </p>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <img
              alt="Picky Eater Logo"
              className="App-logo"
              src="picky-eater-logo.png"
/>,
            "className": "home",
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Picky Eater Logo",
              "className": "App-logo",
              "src": "picky-eater-logo.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              There is a right choice after all.
</p>,
            "className": "tag-line",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "There is a right choice after all.",
            },
            "ref": null,
            "rendered": "There is a right choice after all.",
            "type": "p",
          },
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
